docs/conf.py

```diff
@@ -30,9 +30,51 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.autosummary',
-    #'nbsphinx'
+    # 'breathe',
+    # 'exhale'
 ]
 
+### Breathe + exhale is super slow at the moment. 
+### Keeping this for future implem (save a few hours)
```

Comment:
Just to note here: we tried Breathe+exhale route year ago and it was not "production ready". Tristan is now re-evaluating this after a year. Lets see if we will have positive news on this end.

(not strong opinion on keeping commented section but other option could be to create an issue with this setting so that we have issue on our list as a reminder).

Reply:
Thought no one explored breathe + exhale. In that case I will remove altogether.